x-pack/plugins/synthetics/public/apps/synthetics/utils/testing/__mocks__/syncthetics_store.mock.ts

```diff
@@ -36,6 +37,18 @@ export const mockState: SyntheticsAppState = {
         },
         url: 'https://test.elastic.dev',
         isServiceManaged: true,
+        status: LocationStatus.EXPERIMENTAL,
+      },
+      {
+        id: 'us_central',
```

Comment:
I would expect that this test location should have a different id that the first test location, but I'm not as familiar with these tests. Let's get CI green and revisit.

Reply:
You're right. Will push a fix